src/index.js

```diff
@@ -21,6 +24,16 @@ const run = async () => {
   console.log("->> Generating markdown…");
   const markdown = await generateMarkdownReport(results);
 
+  // Expose the markdown to an Action output
+  const escaped_markdown = markdown.replace(/\%/g,'%25').replace(/\n/g,'%0A').replace(/\r/g,'%0D')
```

Comment:
Is there a documentation reference that can be referenced with this comment?

Reply:
Added!